packages/sdk/src/rpc/types.ts

```diff
@@ -44,6 +44,15 @@ export type StakingPositions = {
   unbonds: Unbonds[];
 };
 
+export type MaspTokenRewards = {
+  name: string;
+  address: string;
+  max_reward_rate: number;
+  kp_gain: number;
+  kd_gain: number;
+  locked_amount_target: number;
+};
+
```

Comment:
only suggestion here would be to map these values to match TS conventions over Rust snake-case, but it's not a huge deal!